app/lib/controlled_work_mappings/cw1.yml

```diff
@@ -0,0 +1,495 @@
+  skip_unless: client_capital_relevant?
+
+# Other property / outstanding mortgage (non-SMOD)
+- name: Other property 1_2
+  type: text
+  source: from_attribute
+  attribute: additional_non_smod_properties_mortgage
+  skip_unless: client_capital_relevant?
+
+# Main home / client's share of property for assessment (non-SMOD)
+- name: undefined_30
+  type: text
+  source: from_attribute
+  attribute: main_home_percentage_owned
+  skip_if: house_in_dispute
+  skip_unless: client_capital_relevant?
+
+# Other property / client's share of property for assessment (non-SMOD)
+- name: undefined_31
+  type: text
+  source: from_attribute
+  attribute: additional_non_smod_properties_percentage_owned
+  skip_unless: client_capital_relevant?
+
+# Main home / total net equity (non-SMOD)
+- name: undefined_32
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: assessment.capital.capital_items.properties.main_home.net_value
+  skip_if: house_in_dispute
+  skip_unless: client_capital_relevant?
+
+# Other property / total net equity (non-SMOD)
+- name: undefined_33
+  type: text
+  source: from_attribute
+  attribute: additional_non_smod_properties_net_value
+  skip_unless: client_capital_relevant?
+
+# Main home / client's share of total net equity (non-SMOD)
+- name: undefined_34
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: assessment.capital.capital_items.properties.main_home.net_equity
+  skip_if: house_in_dispute
+  skip_unless: client_capital_relevant?
+
+# Other property / client's share of total net equity (non-SMOD)
+- name: undefined_35
+  type: text
+  source: from_attribute
+  attribute: additional_non_smod_properties_net_equity
+  skip_unless: client_capital_relevant?
+
+# Main home / final assessed amount of client's equity (non-SMOD)
+- name: undefined_36
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: assessment.capital.capital_items.properties.main_home.assessed_equity
+  skip_if: house_in_dispute
+  skip_unless: client_capital_relevant?
+
+# Other property / final assessed amount of client's equity (non-SMOD)
+- name: undefined_37
+  type: text
+  source: from_attribute
+  attribute: additional_non_smod_properties_assessed_equity
+  skip_unless: client_capital_relevant?
+
+# Savings (non-SMOD)
+- name: undefined_38
+  type: text
+  source: from_attribute
+  attribute: savings
+  skip_if: savings_in_dispute?
+
+# Partner savings (non-SMOD)
+- name: undefined_39
+  type: text
+  source: from_attribute
+  attribute: partner_savings
+
+# Investments (non-SMOD)
+- name: undefined_40
+  type: text
+  source: from_attribute
+  attribute: investments
+  skip_if: investments_in_dispute?
+
+# Partner investments (non-SMOD)
+- name: undefined_41
+  type: text
+  source: from_attribute
+  attribute: partner_investments
+
+# Valuables (non-SMOD)
+- name: undefined_42
+  type: text
+  source: from_attribute
+  attribute: valuables
+  skip_if: valuables_in_dispute?
+
+# Valuables (non-SMOD)
```

Comment:
Partner valuables - just spotted this whilst doing a bit of copy pasta

Reply:
Good spot! Fixed.